### ISSUE TITLE
warning: Mix compiler :phoenix_sass was supposed to return {:ok | :noop | :error, [diagnostic]} but it returned {:noop}

### ISSUE THREAD

**Issue by OldhamMade** [OldhamMade/phoenix_sass#1]
```
warning: Mix compiler :phoenix_sass was supposed to return {:ok | :noop | :error, [diagnostic]} but it returned {:noop}
```

**Comment by OldhamMade** [comment]
Resolved in version 1.5

**Comment by OldhamMade** [comment]
Seems to have returned when used via a Github Action. Still to be investigated.
